package.json

```diff
@@ -16,8 +16,9 @@
     "prepare": "husky install",
     "release": "changeset publish && npm run format",
     "search": "rocket search",
-    "setup": "npm run setup:ts-configs && npm run setup:patch && npm run setup:postinstall",
+    "setup": "npm run setup:ts-configs && npm run setup:patch && npm run setup:playwright && npm run setup:postinstall",
     "setup:patch": "npx patch-package",
+    "setup:playwright": "npx playwright install",
```

Comment:
Why was this required?

Reply:
For some reason Playwright changed their package to no longer install the browser instances with their package and now they need to be manually installed. This includes it as part of the setup. I can remove it, but nevs will need to run `npx playwright install` after each installation to ensure the browser instances are in sync the Playwright version.